Cargo.toml

```diff
@@ -25,3 +25,4 @@ lazy_static = "1.5.0"
 regex = "1.11.1"
 directories = "6.0.0"
 tree-sitter-traversal2 = "0.2.1"
+topiary-core = { git = "https://github.com/tweag/topiary", version = "0.6.0" }
```

Comment:
See https://github.com/tweag/topiary/pull/992#issuecomment-2899127596 for current versioning blockers